scripts/install-cni.sh

```diff
@@ -15,6 +15,9 @@
 # limitations under the License.
 
 set -u -e
+
+echo "netd version: @VERSION@"
```

Comment:
This is fine, but FYI the other substitution variables below use a single @ sign before the name.